chromatic.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "projectId": "Project:66c421d9ea5f69287ba33241",
```

Comment:
please remove, this locks this to qld-gov-au chromatic instead of anyones forked version.

Reply:
chromatic.config.js added to .gitignore